test/unit/test-layout.cxx

```diff
@@ -105,12 +102,12 @@ TEST(LayoutTest, View)
 {
   int* data = new int[10];
 
-  using layout = RAJA::OffsetLayout<int, RAJA::PERM_I, int>;
+  using layout = RAJA::OffsetLayout<>;
 
   /*
    * View is constructed by passing in the layout.
    */
-  RAJA::View<int, layout> view(data, layout({1}, {10}));
+  RAJA::View<int, layout> view(data, layout({1}, std::array<int, 1>{10}));
```

Comment:
A side-note on this topic by the way.  The bare initializer lists with only one element were actually causing breakage in an older version clang I keep around on  one of my macs.  It's clearly not causing notable problems since it passes travis, but it might be worth thinking of a good way to avoid the need for that.

Also, documenting that the OffsetLayout uses *inclusive* rather than *exclusive* upper bounds might be good.  That one really confused me when one of the tests was dying with this branch until I caught it.